website/blog/2025-11-24-aks-automatic-managed-system-node-pools/index.md

```diff
@@ -0,0 +1,188 @@
+
+<!-- truncate -->
+
+:::info
+
+Learn more in the official documentation: [Managed system node pools on AKS Automatic (preview)](https://learn.microsoft.com/azure/aks/automatic/aks-automatic-managed-system-node-pools-about)
+
+:::
+
+## Why it matters
+
+- **Reduced operational overhead:** AKS handles provisioning, patching, upgrades, and scaling for the system pool, so you spend less time on infrastructure maintenance.
+- **Managed add-on hosting at lower cost:** Core services like Azure Monitor agents, CoreDNS, KEDA, VPA, Konnectivity, Eraser, and Metrics Server run on Microsoft-owned infrastructure. Some add-ons and `DaemonSets` still run on nodes in your subscription.
+- **Built-in security policies:** Deployment Safeguards enforce pod security standards, restrict access to platform namespaces, and block risky configurations by default.
+- **Automatic upgrades:** AKS keeps platform components current, reducing the risk of running outdated or vulnerable system software.
+
+![Architecture diagram showing managed system node pools hosted on Microsoft infrastructure with platform components separated from user workloads](aks-managed-arch.svg)
+
+## Components running on managed system node pools
+
+AKS manages the following platform components on the managed system node pool. You don't need to provision capacity for these services.
+
+| Component | Description |
+| --- | --- |
+| [Azure Monitor](https://learn.microsoft.com/azure/aks/monitor-aks) | Collects logs, metrics, and Kubernetes state for observability dashboards and alerts |
+| [CoreDNS](https://learn.microsoft.com/azure/aks/coredns-custom) | Provides cluster DNS resolution for service discovery |
+| [Eraser](https://learn.microsoft.com/azure/aks/image-cleaner) | Removes unused and vulnerable container images from nodes |
+| [KEDA](https://learn.microsoft.com/azure/aks/keda-about) | Scales workloads based on event-driven metrics such as queue length or HTTP traffic |
+| Konnectivity | Maintains secure connectivity between the control plane and nodes |
+| [Metrics Server](https://learn.microsoft.com/azure/aks/monitor-aks-reference) | Exposes resource metrics for Horizontal Pod Autoscaler and kubectl top |
+| [VPA](https://learn.microsoft.com/azure/aks/vertical-pod-autoscaler) | Recommends and applies optimal CPU and memory requests for pods |
+| [Workload Identity](https://learn.microsoft.com/azure/aks/workload-identity-overview) | Enables pods to authenticate to Azure services using Microsoft Entra ID |
+
+Other add-ons and extensions run on `aks-system-surge` nodes, with scaling handled by [Node Auto-Provisioning (NAP)](https://learn.microsoft.com/azure/aks/node-auto-provisioning). `DaemonSets` run on both managed system node pools and nodes in your subscription.

```

Comment:
can we clarify which ones with a table. For example, as a user I was thinking 'where would these run - Azure Policy Istio (within that Istio control plane vs ingress vs sidecar), ACNS, Microsoft Defender for Containers...? if we have that in the docs, referencing that would suffice too

Reply:
Anything else outside of the list above will run on the surge nodes. They're too many to list.
The list above is in docs already